WoWPro/WoWPro_ZoneData.lua

```diff
@@ -98,7 +98,7 @@ DefineZone9(0008, "Palemane Rock", UIMapType.Micro, 0007, nil)
 DefineZone9(0009, "The Venture Co. Mine", UIMapType.Micro, 0007, nil)
 DefineZone9(0010, "Northern Barrens", UIMapType.Zone, 0012, nil, 0011, 0092, 0279, 0825, 0859, 1339)
 DefineZone9(0011, "Wailing Caverns", UIMapType.Micro, 0010, nil)
-DefineZone9(0012, "Kalimdor", UIMapType.Continent, 0947, nil, 0001, 0007, 0010, 0062, 0063, 0064, 0065, 0066, 0069, 0070, 0071, 0076, 0077, 0078, 0080, 0081, 0083, 0085, 0088, 0097, 0103, 0106, 0198, 0199, 0249, 0327, 0524, 0775, 0776, 0782, 0785, 0891, 0907, 1203, 1332, 1333, 1338, 1469, 1527, 1534, 1535)
+DefineZone9(0012, "Kalimdor", UIMapType.Continent, 0947, nil, 0001, 0007, 0010, 0062, 0063, 0064, 0065, 0066, 0069, 0070, 0071, 0076, 0077, 0078, 0080, 0081, 0083, 0085, 0088, 0097, 0103, 0106, 0198, 0199, 0327, 0524, 0775, 0776, 0782, 0785, 0891, 0907, 1203, 1332, 1333, 1338, 1469, 1527, 1534, 1535)
```

Comment:
This dropped 0249, aka Uldum.  Not good.

Reply:
Crap I thought I corrected that one.